cedar-policy/src/tests.rs

```diff
@@ -6230,6 +6231,31 @@ mod reserved_keywords_in_policies {
         "unless",
     ];
 
+    lazy_static! {
+        static ref FUNCTION_SUGGESTIONS: HashMap<&'static str, &'static str> = {
+            let mut map = HashMap::with_capacity(OTHER_SPECIAL_IDENTS.len());
+            map.insert("principal", "decimal");
+            map.insert("action", "decimal");
+            #[cfg(not(feature = "experimental"))]
+            {
+                map.insert("resource", "decimal");
```

Comment:
Just a thought, but perhaps there should be some similarity cut-off so that we don't have this sort of more or less useless suggestion.

Reply:
Yes, I was thinking the same. And now that I've wasted an hour with this testing code, I'm fully convinced we should it here.